src/ralph/accessories/tests/factories.py

```diff
@@ -0,0 +1,32 @@
+from factory import post_generation, Sequence, SubFactory
+from factory.django import DjangoModelFactory
+
+from ralph.accessories.models import Accessory, AccessoryStatus, AccessoryUser
+from ralph.accounts.tests.factories import RegionFactory, UserFactory
+from ralph.assets.tests.factories import CategoryFactory, ManufacturerFactory
+from ralph.back_office.tests.factories import WarehouseFactory
+
+
+class AccessoryFactory(DjangoModelFactory):
```

Comment:
This is different than data center Accessory. 
It wasn't tested at all.
There is also no API